spec/suites/layer/LayerGroupSpec.js

```diff
@@ -112,4 +112,64 @@
 			expect(layers[1].options.opacity).to.eql(opacity);
 		});
 	});
+
+	describe('#getBounds', () => {
+		it('returns the bounds (LatLngBounds) of the group', () => {
+			const southWest = L.latLng(0, 0),
+			northEast = L.latLng(4, 4);
+
+			const fg = L.featureGroup([
+				L.marker(southWest),
+				L.marker(northEast),
+				L.marker([3, 3]),
+			]);
+
+			const bounds = L.latLngBounds(southWest, northEast);
+			expect(fg.getBounds()).to.eql(bounds);
+		});
+
+		it('doesn\'t break if there are nested layer groups and returns the bounds of the nested groups', () => {
+			const southWest = L.latLng(0, 0),
+			northEast = L.latLng(4, 4);
+
+			const childGroup = L.featureGroup([
+				L.marker(southWest),
+				L.marker(northEast),
+			]);
+
+			const parentGroup = L.layerGroup([
+				childGroup
+			]);
+
+			const bounds = L.latLngBounds(southWest, northEast);
+
+			let result = [];
+			const callback = () => {
+				result = parentGroup.getBounds();
+			};
+			expect(callback).to.not.throwError();
+			expect(result).to.eql(bounds);
+		});
+	});
```

Comment:
This works with the current solution too, you need to add to the `childGroup` the `parenGroup`: `childGroup.addLayer(parentGroup)`

Reply:
Oops, forgot to add it. My bad.